docs/tutorials/mediator.md

```diff
@@ -118,17 +118,18 @@ There isn't much to this code -- and that's the entire point! When Wolverine reg
 a .NET Core application, it adds the `IMessageBus` service to the underlying system IoC container
 so it can be injected into controller classes or Minimal API endpoint as shown above.The `IMessageBus.InvokeAsync(message)`
 method takes the message passed in, finds the correct execution path for the message type, and
-executes the correct Wolverine handler(s) as well as any of the registered [Wolverine middleware](/guide/messages/middleware).
+executes the correct Wolverine handler(s) as well as any of the registered [Wolverine middleware](/guide/handlers/middleware).
 
 ::: tip
-This execution happens inline, but will use the *RetryNow* error handling capabilities. See [Wolverine's error handling](/guide/messages/error-handling) for more information.
+This execution happens inline, but will use the *RetryNow* error handling capabilities. See [Wolverine's error handling](/guide/handlers/error-handling) for more information.
 :::
 
 See also:
 
-* [Cascading messages from actions](/guide/messages/handlers.html#cascading-messages-from-actions) for a better explanation of how the `ItemCreated`
+* [Cascading messages from actions](/guide/handlers/cascading) for a better explanation of how the `ItemCreated`
   event message is automatically published if the handler success.
-* [Messages and message handlers](/guide/messages/) for the details of how to write Wolverine message handlers and how they
+* [Messages](/guide/messages) for the details of messages themselves including versioning, serialization, and forwarding.
```

Comment:
Added another bullet point for messages themselves as this section appears to now be broken up into 2.